src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/domain/BehandlingRepository.kt

```diff
@@ -62,6 +62,7 @@ interface BehandlingRepository :
               b.resultat,
               b.vedtakstidspunkt,
               b.henlagt_arsak,
+              b.henlagt_begrunnelse,
```

Comment:
Kan være fint å sjekke at testen i BehandlingRepositoryTest -> finnSaksbehandling får med begrunnelsen då det er en custom query for dette

Reply:
Fikser!